src/Entity/PrintArticle.php

```diff
@@ -257,23 +210,25 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
         'text_processing' => 0,
       ])
       ->setDefaultValue('')
-      ->setDisplayOptions('view', [
-        'label' => 'above',
-        'type' => 'string',
-        'weight' => -4,
-      ])
       ->setDisplayOptions('form', [
         'type' => 'string_textfield',
         'weight' => -4,
       ])
-      ->setDisplayConfigurable('form', TRUE)
-      ->setDisplayConfigurable('view', TRUE);
+      ->setDisplayConfigurable('form', TRUE);
 
     $fields['status'] = BaseFieldDefinition::create('boolean')
       ->setLabel(t('Publishing status'))
       ->setDescription(t('A boolean indicating whether the Print article is published.'))
```

Comment:
That is the label and description provided to the user. So we should change that to something like
* Label: _Published_
* Description: _Check to make this article available for download ..._